src/alliteratedActors.js

```diff
@@ -22,7 +22,10 @@
  * @return {string}
  */
 function alliteratedActors(names) {
-  // write code here
+  return names.find(el => {
+    let fullNames = el.split(' ');
+    return fullNames.every(el => el.includes(fullNames[0][0]));
```

Comment:
`el` is already declared in the upper scope. Please avoid such kind of overriding.
It will be better to define some variable `const letter = fullNames[0][0]` before the loop `every` to prevent requesting it on every iteration